src/components/Profile.tsx

```diff
@@ -62,6 +86,8 @@ const Profile: React.FC = () => {
         max-width: 820px;
       `}
     >
+      <Button onClick={toggleDarkMode}>Toggle Dark Mode</Button>
+      <Button onClick={resetToSystem}>Reset to System Theme</Button>
```

Comment:
```suggestion
  {window.darkMode && <>
      <Button onClick={toggleDarkMode}>Toggle Dark Mode</Button>
      <Button onClick={resetToSystem}>Reset to System Theme</Button>
   </>}
```

Only show these buttons if `darkMode` is defined.